src/App/screens/Instructor/screens/NewLessons/components/Submit/index.js

```diff
@@ -90,19 +112,44 @@ export default connect(
         </div>
 
         <div className='mb2'>
+          <div className='b gray'>
+            {lessonTitleLabelText}
+          </div>
           <input
             type='text'
-            placeholder={lessonTitleLabelText}
             value={title}
             onChange={this.handleTitleChange}
             className={`${inputClassNames}${hasError ? ' b--red' : ''}`}
           />
         </div>
 
         <div className='mb2'>
+          <div className='b gray'>
+            {lessonTechnologyLabelText}
+          </div>
+          <select
+            value={technologyId}
+            onChange={this.handleTechnologyChange}
+            className={`${inputClassNames}${hasError ? ' b--red' : ''}`}
+          >
+            <option value=''></option>
+            {map(this.props.technologies, (technology) => (
+              <option 
+                key={technology.id}
+                value={technology.id}
+              >
+                {technology.label}
+              </option>
+            ))}
+          </select>
+        </div>
+
+        <div className='mb2'>
+          <div className='b gray'>
+            {lessonSummaryLabelText}
+          </div>
           <textarea
             type='text'
-            placeholder={lessonSummaryLabelText}
             rows='5'
             value={summary}
             onChange={this.handleSummaryChange}
```

Comment:
Some of this is logic that could be consolidated with the other form in the app, but I'm waiting for the styleguide components to be published so I can consume and update them; the duplication here is ok until then. cc: @tayiorbeii